testdriver/testplan.py

```diff
@@ -201,7 +201,7 @@ def generate_header(self):
             # These should come from the Executor
             "icu_version": '%s' % self.icuVersion,
             "cldr_version": '%s' % self.cldrVersion,
```

Comment:
similar change as the line below to remove the redundant `'%s' % ...`?

Reply:
Done